src/com/seafile/seadroid2/account/Account.java

```diff
@@ -10,31 +10,20 @@
 public class Account implements Parcelable, Comparable<Account> {
     private static final String DEBUG_TAG = "Account";
 
+    /**
+     * Type of the account (currently there is only one type)
+     */
+    public final static String ACCOUNT_TYPE = "com.seafile.seadroid2";
```

Comment:
I think it\`s better to append more strings, e.g. "com.seafile.seadroid2.account.type"